### ISSUE TITLE
`source` mark for `"curly repack"` should exclude braces

### ISSUE THREAD

**Issue by pokey** [cursorless-dev/cursorless#744]
In the test `src/test/suite/fixtures/recorded/actions/curlyRepackRound.yml`, the source mark should exclude the curly braces but it does not.

**Comment by AndreasArvidsson** [comment]
Not quite sure. For wrap we are wrapping the thing you're actually targeting. The input target to the wrap action becomes the source target and for rewrap we do the same thing so in that sense they are consistent. The different is that you can't wrap something without first targeting it, but you can do that with rewrap.

**Comment by pokey** [comment]
Sure, but philosophical arguments aside, isn't it more useful to have "source" be different from "target" here?

**Comment by AndreasArvidsson** [comment]
Or source should be the thing you actually targeted and that should be the full pair?

**Comment by pokey** [comment]
I don't think so, because in most actions, "that" is the thing you targeted.  Tbh if anything I'd be tempted to change "that" to be the interior for wrap to make things more consistent with other actions, as that's what you targeted.  But it's prob a bit late on that one.  Either way I think most consistent / useful for now is to make "source" be interior for rewrap, the same way it is for "wrap"
